src/svm.rs

```diff
@@ -315,6 +317,8 @@ struct RawValIter<T> {
     end: *const T,
 }
 
+unsafe impl<T: Send> Send for RawValIter<T> {}
```

Comment:
As above, this is also a part of the `SvmVec` implementation,  but `SvmVec` is not `Send`, is this correct?

Reply:
See https://github.com/kenba/opencl3/pull/16#discussion_r641981784.